src/features/island/hud/components/inventory/Chest.tsx

```diff
@@ -70,86 +71,78 @@ export const Chest: React.FC<Props> = ({
     getKeys(collectibles)[0]
   );
 
-  const handlePlace = () => {
-    onPlace && onPlace(selected);
-
-    closeModal();
-  };
-
-  const handleItemClick = (item: InventoryItemName) => {
-    setSelected(item);
-  };
-
   const chestIsEmpty = getKeys(collectibles).length === 0;
-
   if (chestIsEmpty) {
     return (
       <div className="flex flex-col justify-evenly items-center p-2">
-        <img src={chest} className="h-12" alt="Empty Chest" />
-        <span className="text-xs text-center mt-2 w-80">
+        <img
+          src={chest}
+          alt="Empty Chest"
+          style={{
+            width: `${PIXEL_SCALE * 17}px`,
+          }}
+        />
+        <span className="text-xs text-center mt-2">
           Your chest is empty, discover rare items today!
         </span>
       </div>
     );
   }
 
+  const handlePlace = () => {
+    onPlace && onPlace(selected);
+
+    closeModal();
+  };
+
+  const handleItemClick = (item: InventoryItemName) => {
+    setSelected(item);
+  };
+
   return (
     <SplitScreenView
       divRef={divRef}
       tallMobileContent={true}
       wideModal={true}
-      showHeader={!chestIsEmpty && !!selected}
+      showHeader={!!selected}
       header={
         selected && (
-          <>
-            <div className="flex flex-col justify-center p-2 pb-0">
-              <div className="flex space-x-2 justify-start mb-1 items-center sm:flex-col-reverse md:space-x-0">
-                <div className="sm:mt-2">
-                  <SquareIcon icon={ITEM_DETAILS[selected].image} width={14} />
-                </div>
-                <span className="sm:text-center">{selected}</span>
-              </div>
-              <span className="text-xs sm:text-center">
-                {ITEM_DETAILS[selected].description}
-              </span>
-              <div className="border-t border-white w-full my-2 pt-1 flex justify-between sm:flex-col sm:items-center">
-                <a
-                  href={`https://opensea.io/assets/matic/0x22d5f9b75c524fec1d6619787e582644cd4d7422/${KNOWN_IDS[selected]}`}
-                  className="underline text-xxs hover:text-blue-500 p-2"
-                  target="_blank"
-                  rel="noopener noreferrer"
-                >
-                  OpenSea
-                </a>
-              </div>
-            </div>
-            {onPlace && (
-              <Button onClick={handlePlace} disabled={isSaving}>
-                {isSaving ? "Saving..." : "Place on map"}
-              </Button>
-            )}
-          </>
+          <CraftingRequirementsView
```

Comment:
This is a case where I'm not sure it makes sense that a `readonly` chest view uses a `crafting` view to display it's details.

Reply:
A different component is now used